docs/guide/en/displaying-fields.md

```diff
@@ -9,6 +9,13 @@ use Yiisoft\FormModel\FormModel;
 
 /** @var FormModel $formModel */
 $field = Field::text($formModel, 'login');
+
+/** Display fields nested forms */
```

Comment:
```suggestion
/** Display nested form */
```
Does that display whole form?

Reply:
Whole?

nested form field or nested form 🤷‍♂️